lesson_04/expression/expression_app/src/main/java/com/codedifferently/lesson4/ExpressionCalculator.java

```diff
@@ -3,10 +3,33 @@
 public class ExpressionCalculator {
   /**
    * Returns the calculation of ((a + b) * c) / d^e
+   *
+   * @param a
```

Comment:
Next time, make sure you sync so that you don't re-add things that I've removed :D